src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -53,8 +53,6 @@ public SolutionChecksumUpdater(Workspace workspace, IGlobalOptionService globalO
             Start();
         }
 
-        private CancellationToken ShutdownCancellationToken => CancellationToken;
```

Comment:
extremely confusing to have two CTs in scope that are the same thing, but with different names, making it unclear what it means when some code in this type refers to one, and some to the other.